src/main/java/com/glencoesoftware/bioformats2raw/Converter.java

```diff
@@ -251,6 +253,13 @@ public class Converter implements Callable<Void> {
     BioTekReader.class, ND2PlateReader.class
   };
 
+  @Option(
+          names = "--no-omero", negatable=true,
```

Comment:
As you mention, with an eye to https://github.com/ome/ngff/issues/78, is use of "omero" here intended to allow deprecation in favor of "--no-rendering" in the future or would it be worth considering to use the future terminology now?

Reply:
`--[no-]minmax` would have the benefit of being consistent with the option in the Bio-Formats command-line tools - see `showinf -h`